packages/dev/gui/src/2D/controls/sliders/scrollBar.ts

```diff
@@ -45,6 +51,20 @@ export class ScrollBar extends BaseSlider {
         this._markAsDirty();
     }
 
+    /** Gets or sets background gradient. Takes precedence over gradient. */
+    public get backgroundGradient(): Nullable<BaseGradient> {
+        return this._backgroundGradient;
+    }
+
+    public set backgroundGradient(value: Nullable<BaseGradient>) {
+        if (this._backgroundGradient === value) {
+            return;
+        }
+
+        this._backgroundGradient = value;
+        this._markAsDirty();
+    }

```

Comment:
Adding this to the BaseSlider (or a higher level class?) will save a bit of repeated code. Part of the code doc should be that it will be ignored where it doesn't make sense

Reply:
I think I left them in both ScrollBar and Slider to avoid adding it to ImageBasedSlider too, but if it's not an issue to have it there even through it's not used, I can add it?